code/datums/jobs.dm

```diff
@@ -19,6 +19,7 @@
 	var/recieves_miranda = 0
 	var/recieves_implant = null //Will be a path.
 	var/receives_disk = 0
+	var/receives_security_disk = 0
```

Comment:
unused?

Reply:
Oops, forgot to actually check this flag when adding the SECAUTH file to the disk! It's fixed now.